src/components/search-results/searchResult.tsx

```diff
@@ -0,0 +1,37 @@
+import styles from './searchResult.module.css';
+import SearchCard from '../search-card/SearchCard';
+import { SpellsRequestData } from '../../types/requests-types';
+import { Link, useLocation } from 'react-router-dom';
+
+function SearchResult(props: { peopleRequest: SpellsRequestData[] }) {
+  const location = useLocation();
+  const results = props.peopleRequest.map((el) => {
+    return (
+      <div key={el.id} className={styles.searchCard}>
+        <Link
+          to={`/details/${el.id}`}
+          className={styles.searchCard}
+          onClick={(event) => {
+            if (location.pathname !== '/') {
+              event.preventDefault();
```

Comment:
You can create function outside the loop and use 1 function for each component

Reply:
Done ✅